frog/imports/ui/GraphEditor/SidePanel/OperatorPanel/EditOperator.jsx

```diff
@@ -45,6 +47,18 @@ const TopPanel = ({
         }}
       >
         <ValidButton activityId={operator._id} errorColor={errorColor} />
+        {operatorType.meta.preview && (
+          <IconButton
+            icon="glyphicon glyphicon-eye-open"
+            tooltip="Preview"
+            onClick={() => {
+              ui.setShowPreview({
+                operatorTypeId: operatorType.id,
+                config: operator.data
+              });
+            }}
+          />
```

Comment:
This button should be disabled if the operator is not valid. Right now it crashes on Hypothesis

Reply:
Good point. Why does it crash though - is it if you don't type anything in any of the fields?

Reply:
Fixed